lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -143,23 +143,62 @@ function flashNotice(type, notice, button_label, button_function, button_type) {
   if (type == 'success') { setTimeout(function() {$('#gollum-flash').fadeOut();}, 5000); }
 }
 
+// Check whether the user has allowed their browser to access their system
+// clipboard.
+async function isClipboardAccessDenied() {
+  if (navigator.permissions && navigator.permissions.query) {
```

Comment:
It would seem that nothing is returned when this `if` statement fails (e.g., if `navigator.permissions` does not exist). Is this the desired behavior or do we need an additional `else` clause?

Reply:
For me, the function name `isClipboardAccessDenied` signals a boolean response type. Either access is denied or not. Calling this function, I would not expect to receive `undefined`. @dometto ?

Reply:
I don't think `undefined` is too strange a result in case the permissions aren't set, but I can see how the method name could be a bit confusing. What about changing it to something more neutral, e.g. `checkClipboardAccessDenied`?

Reply:
Could we rename it `checkClipboardAccess` instead and flip the return statement? The comment also says "Check whether the user has allowed their browser to access their system".

Reply:
Otherwise I'm happy with `checkClipboardAccessDenied` and I'll shut up. Thanks for your patience @benjaminwil !

Reply:
Thanks for helping me resolve! I just pushed a fix to this commit with the new name.